tms/__main__.py

```diff
@@ -0,0 +1,144 @@
+"""Entrypoint for TMS."""
+
+
+import asyncio
+import logging
+from pathlib import Path
+from typing import Any
+
+import htcondor  # type: ignore[import-untyped]
+from rest_tools.client import RestClient
+
+from . import utils
+from .config import ENV, config_logging
+from .starter import starter
+from .stopper import stopper
+from .watcher import watcher
+
+LOGGER = logging.getLogger(__name__)
+
+
+async def starter_loop() -> None:
+    """Listen to EWMS and start designated taskforces."""
+
+    # make connections -- do now so we don't have any surprises downstream
+    schedd_obj = htcondor.Schedd()  # no auth need b/c we're on AP
+    ewms_rc = RestClient(ENV.EWMS_ADDRESS, token=ENV.EWMS_AUTH)
+    LOGGER.info("Connected to EWMS")
+
+    async def next_one() -> dict[str, Any]:
+        """Get the next taskforce requested for this collector + schedd."""
+        return await ewms_rc.request(  # type: ignore[no-any-return]
+            "GET",
+            "/tms/taskforce/start",
+            {"collector": ENV.COLLECTOR, "schedd": ENV.SCHEDD},
+        )
+
+    while True:
+        while args := await next_one():
+            ewms_taskforce_attrs = await starter.start(
+                schedd_obj,
+                utils.is_taskforce_to_be_aborted(ewms_rc, args["taskforce_uuid"]),
+                **args,  # TODO
+            )
+            # confirm start (otherwise ewms will request this one again -- good for statelessness)
+            await ewms_rc.request(
+                "PATCH",
+                f"/tms/taskforce/start/{args['taskforce_uuid']}",
```

Comment:
This feels a bit wrong to me, since we're patching the start taskforce collection, instead of deleting it here and adding it to a new "running" taskforce collection.  Or if we wanted to do it in one action, maybe it should be a POST to `/tms/taskforce/run` and the API knows to remove from start?

Reply:
I renamed these, see: [`628f68a` (#3)](https://github.com/Observation-Management-Service/ewms-task-management-service/pull/3/commits/628f68a3379aa9e8b59afdea3f438c1be4a6134b)